ProcessLib/ProcessVariable.cpp

```diff
@@ -113,4 +114,24 @@ MeshLib::Mesh const& ProcessVariable::getMesh() const
 	return _mesh;
 }
 
+MeshLib::PropertyVector<double>& ProcessVariable::getOrCreateMeshProperty()
+{
+	boost::optional<MeshLib::PropertyVector<double>&> result;
```

Comment:
Too lazy to check by myself: Why do we need `boost::optional` here? The code below looks like it worked well without it.